src/datastore/StringCrypto.ts

```diff
@@ -29,7 +34,7 @@ export class StringCrypto {
 
             // Test whether key is a valid PEM key (publicEncrypt does internal validation)
```

Comment:
This probably warrants another PR, but should symmetric encryption be used instead of RSA?  Since encryption & decryption are done by the same entity (i.e. the bridge), there doesn't seem to be an advantage to using asymmetric encryption here.

Granted, changing to symmetric encryption would require migrating any existing secrets stored in the database & making `ircService.passwordEncryptionKeyPath` expect some kind of password/secret other than an RSA privkey, so maybe it's not worth the friction at this point.

Reply:
Yeah, I agree. Unfortunately I think that would be a (necessary) breaking change for the moment.